2021-11-21 22:09:40,115 INFO    MainThread:79347 [wandb_setup.py:_flush():71] setting env: {'project': '1018_lw'}
2021-11-21 22:09:40,115 INFO    MainThread:79347 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-21 22:09:40,116 INFO    MainThread:79347 [wandb_init.py:_log_setup():371] Logging user logs to /home/yu-jw19/venom/project2/wandb/run-20211121_220940-2bmtuwhe/logs/debug.log
2021-11-21 22:09:40,116 INFO    MainThread:79347 [wandb_init.py:_log_setup():372] Logging internal logs to /home/yu-jw19/venom/project2/wandb/run-20211121_220940-2bmtuwhe/logs/debug-internal.log
2021-11-21 22:09:40,116 INFO    MainThread:79347 [wandb_init.py:init():404] calling init triggers
2021-11-21 22:09:40,116 INFO    MainThread:79347 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-11-21 22:09:40,117 INFO    MainThread:79347 [wandb_init.py:init():449] starting backend
2021-11-21 22:09:40,117 INFO    MainThread:79347 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-21 22:09:40,143 INFO    MainThread:79347 [backend.py:ensure_launched():199] starting backend process...
2021-11-21 22:09:40,157 INFO    MainThread:79347 [backend.py:ensure_launched():204] started backend process with pid: 79567
2021-11-21 22:09:40,158 INFO    MainThread:79347 [wandb_init.py:init():458] backend started and connected
2021-11-21 22:09:40,160 INFO    MainThread:79347 [wandb_init.py:init():519] updated telemetry
2021-11-21 22:09:40,208 INFO    MainThread:79347 [wandb_init.py:init():549] communicating current version
2021-11-21 22:09:44,333 INFO    MainThread:79347 [wandb_init.py:init():554] got version response 
2021-11-21 22:09:44,333 INFO    MainThread:79347 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-11-21 22:09:44,859 INFO    MainThread:79347 [wandb_init.py:init():592] starting run threads in backend
2021-11-21 22:09:44,906 INFO    MainThread:79347 [wandb_run.py:_console_start():1816] atexit reg
2021-11-21 22:09:44,907 INFO    MainThread:79347 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-11-21 22:09:44,907 INFO    MainThread:79347 [wandb_run.py:_redirect():1695] Redirecting console.
2021-11-21 22:09:44,909 INFO    MainThread:79347 [wandb_run.py:_redirect():1751] Redirects installed.
2021-11-21 22:09:44,909 INFO    MainThread:79347 [wandb_init.py:init():619] run started, returning control to user process
2021-11-21 22:09:44,916 INFO    MainThread:79347 [wandb_run.py:_config_callback():962] config_cb None None {'dataset': 'emotiondataset', 'name': 'first_test_network==44', 'no': '0', 'model': 'resnet', 'print_freq': 10, 'layers': 44, 'droprate': 0.3, 'cardinality': 8, 'growth_rate': 12, 'compression_rate': 0.5, 'bn_size': 4, 'alpha': 200, 'N': 2, 'M': 10, 'augment': True, 'randaugment': True, 'checkpoint': 'checkpoint', 'resume': False, 'cos_lr': True, 'initial_learning_rate': 0.1, 'changing_lr': [80, 120], 'en_wandb': True, 'warm_up': False, 'exp_id': '00013_network==44', 'commit': 'not_set'}
2021-11-21 22:10:13,611 INFO    MainThread:79347 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-11-21 22:10:13,611 INFO    MainThread:79347 [wandb_run.py:_restore():1758] restore
2021-11-21 22:10:16,058 INFO    MainThread:79347 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 970
}

2021-11-21 22:10:16,170 INFO    MainThread:79347 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 970
}

2021-11-21 22:10:16,686 INFO    MainThread:79347 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 970
}

2021-11-21 22:10:16,897 INFO    MainThread:79347 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 8999
}

2021-11-21 22:10:17,001 INFO    MainThread:79347 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 8999
}

2021-11-21 22:10:17,104 INFO    MainThread:79347 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 8999
}

2021-11-21 22:10:17,208 INFO    MainThread:79347 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 8999
}

2021-11-21 22:10:17,311 INFO    MainThread:79347 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 8999
}

2021-11-21 22:10:17,419 INFO    MainThread:79347 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 8999
}

2021-11-21 22:10:17,523 INFO    MainThread:79347 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 8999
}

2021-11-21 22:10:17,627 INFO    MainThread:79347 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8999
  total_bytes: 8999
}

2021-11-21 22:10:17,730 INFO    MainThread:79347 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8999
  total_bytes: 8999
}

2021-11-21 22:10:17,834 INFO    MainThread:79347 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8999
  total_bytes: 8999
}

2021-11-21 22:10:17,937 INFO    MainThread:79347 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8999
  total_bytes: 8999
}

2021-11-21 22:10:18,040 INFO    MainThread:79347 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8999
  total_bytes: 8999
}

2021-11-21 22:10:18,464 INFO    MainThread:79347 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8999
  total_bytes: 8999
}

2021-11-21 22:10:19,532 INFO    MainThread:79347 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8999
  total_bytes: 8999
}
local_info {
}

2021-11-21 22:10:20,631 INFO    MainThread:79347 [wandb_run.py:_append_files():2184] logging synced files
